app/styles/modules/_custom-rows.scss

```diff
@@ -14,6 +14,10 @@
     margin: 20px 0 15px 0;
   }
 
+  &.margin-top {
```

Comment:
I tried that originally and it made them go blue. They're not blue in the mockup. Did I do it wrong?